CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
 === Develop locally on your host
 
 You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker
-installed on Linux to actually launch containers.
+installed on Linux to actually launch containers. Client and server binaries
+can be built locally or in the `openshift/release` container environemnt. The
+http://golang.org/[Go] programming language is only necessary for building on
+the local host.
```

Comment:
@jcajka I can't say I remember having to run `go install -race std` on our RHEL/CentOS systems to get the race detector to work. Is this different in Fedora? On the  RHEL/CentOS test systems we just `yum install golang` and we get everything like so:

```
Loaded plugins: amazon-id, rhui-lb, search-disabled-repos
Resolving Dependencies
--> Running transaction check
---> Package golang.x86_64 0:1.7.5-1.1.el7 will be installed
--> Processing Dependency: golang-src = 1.7.5-1.1.el7 for package: golang-1.7.5-1.1.el7.x86_64
--> Processing Dependency: golang-bin = 1.7.5-1.1.el7 for package: golang-1.7.5-1.1.el7.x86_64
--> Running transaction check
---> Package golang-bin.x86_64 0:1.7.5-1.1.el7 will be installed
---> Package golang-src.noarch 0:1.7.5-1.1.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package
   Arch   Version
             Repository                                                    Size
================================================================================
Installing:
 golang             x86_64         1.7.5-1.1.el7         xxx               1.2 M
Installing for dependencies:
 golang-bin         x86_64         1.7.5-1.1.el7         xxx                43 M
 golang-src         noarch         1.7.5-1.1.el7         xxx               4.4 M

Transaction Summary
================================================================================
Install  1 Package (+2 Dependent packages)

Total download size: 49 M
Installed size: 204 M
Downloading packages:
--------------------------------------------------------------------------------
Total                                               22 MB/s |  49 MB  00:02     
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : golang-src-1.7.5-1.1.el7.noarch                              1/3 
  Installing : golang-1.7.5-1.1.el7.x86_64                                  2/3 
  Installing : golang-bin-1.7.5-1.1.el7.x86_64                              3/3 
  Verifying  : golang-bin-1.7.5-1.1.el7.x86_64                              1/3 
  Verifying  : golang-src-1.7.5-1.1.el7.noarch                              2/3 
  Verifying  : golang-1.7.5-1.1.el7.x86_64                                  3/3 

Installed:
  golang.x86_64 0:1.7.5-1.1.el7                                                 

Dependency Installed:
  golang-bin.x86_64 0:1.7.5-1.1.el7      golang-src.noarch 0:1.7.5-1.1.el7     

Complete!
```